.gitignore

```diff
@@ -1,4 +1,5 @@
 logfile.txt
+rpgtools/
```

Comment:
```suggestion

```

Reply:
yeah it's not necessary as git knows is a submodule